### ISSUE TITLE
App crashes when switching.

### ISSUE THREAD

**Issue by anidh** [devRant/devRant#475]
When we are on upload a **new joke/meme screen** and when we have already uploaded a file and written some description [See image (1)] and when we press the app switcher button the app immediately crashes. I'm using the **version 1.12.1.1** on my **Oneplus 5**

![screenshot_20180720-192119](https://user-images.githubusercontent.com/17473554/43006263-e13b9afc-8c52-11e8-8d01-c4e9a3a28e23.jpg)
In this image, the image is already selected and some description is already added. Now when we click on the phone's recent app button.
![screenshot_20180720-192155](https://user-images.githubusercontent.com/17473554/43006358-1ecb02f4-8c53-11e8-8cb7-ba28f54f4aa2.jpg)
As we can see the app has now crashed. I tried to replicate the bug 3-4 times and every time the bug popped up.

**Comment by Skayo** [comment]
Cannot reproduce. I'm using a Huawei P10, same devRant Version.
Probably Phone/OS specific...

**Comment by anidh** [comment]
Have You tried above steps?

- Open devRant App
- Now Goto Add **Rant/Story**
- Then Add some random description and attach an image **don't** add any tags and **don't** post it 
   yet
- Now press the app switcher button on your phone and the app crashes.

Just follow the above steps once more.

**Comment by Skayo** [comment]
@anidh Yes, tried that.
Still no crash.

**Comment by anidh** [comment]
Maybe it's an OS/Model specific problem. Can you try to reproduce the issue on OnePlus 5 with lastest android update or anyone who can do the same.

**Comment by Skayo** [comment]
Unfortunately I do not have a OnePlus 5.
Let's see if someone sees this...

**Comment by christoph-fricke** [comment]
@anidh I am using an OP5 and I was not able to reproduce. Latest Software versions (stable branch). I use the hardware button for app switching.

**Comment by anidh** [comment]
@christoph-fricke Have you pressed the app switch button when you have attached an image. People using the same OS/Model have reported the problem. Can you try once again? Remember to attach an image and then try to switch app. If it doesn't work then maybe try with some description inserted and image attached and then try to multitask.

**Comment by anidh** [comment]
@nicoco007 Yeah i'm using android 8.1.0 stable build. I think the error is specific to the Oxygen OS.

**Comment by anidh** [comment]
@queenCee Seems to be an Oxygen OS issue. @Skayo This issue is persistent across whole Oneplus family imo seems to be Oxygen OS issue.

**Comment by Skayo** [comment]
@DFoxinator you think you can fix this bug?
Or is it a non-devRant problem

**Comment by christoph-fricke** [comment]
@anidh I am using OxygenOS 5.1.3 and I tried exactly what you described.
- Click Add button
- New Joke/Meme (Also tried Rant/Story)
- Add a description
- Add an image (I tried PNG & JPG)
- Click App switcher button
- Switch to another App
- Come back

And it still works smoothly...

**Comment by Skayo** [comment]
Btw I've added the wontfix because I'm quite sure it's not a problem of devRant.

**Comment by rEaL-jAsE** [comment]
This happens to me on Nexus 6 and 6P and Pixel phones running every version of Android that has been installed on them since I started using devRant.

**Comment by Skayo** [comment]
Oh what the heck now I can reproduce. Not sure what I did wrong the first time... (Sorry btw)

Guess this is pretty high priority now @DFoxinator

**Comment by anidh** [comment]
@Skayo any changes from the last time when you weren't able to reproduce the bug? I still face this issue.
